src/stores/BallotStore.js

```diff
@@ -22,7 +22,9 @@ class BallotStore {
 		2: 'VotingToChangeKeys',
 		3: 'VotingToChangeMinThreshold',
 		4: 'VotingToChangeProxy',
-		5: 'BallotsStorage'
+		5: 'BallotsStorage',
+		7: 'ValidatorMetadata',
+		8: 'ValidatorMetadataEternalStorage'
```

Comment:
I don't get it, can we change the storage? How?

Reply:
Yes, we can change the address of **ValidatorMetadata** storage. See **setContractAddress** function in **ProxyStorage.sol**. This function is called by **finalizeBallot** in **VotingToChangeProxyAddress.sol**.